tests/ut/ir/transforms/test_deep_clone.py

```
@@ -0,0 +1,154 @@
+    def test_clone_assign_stmt(self):
+        """Deep clone creates fresh Var for AssignStmt LHS."""
+
+        @pl.program
+        class P:
+            @pl.function
+            def main(self, x: pl.Tensor[[64], pl.FP32]) -> pl.Tensor[[64], pl.FP32]:
+                y: pl.Tensor[[64], pl.FP32] = pl.create_tensor([64], dtype=pl.FP32)
+                return y
+
+        func = _get_function(P, "main")
+        cloned_body, var_map = ir.deep_clone(func.body)
```

Ruff (0.15.5)

[warning] Unpacked variable `cloned_body` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)